### PR TITLE
Misc. needed for jack

### PR DESCRIPTION
**Pull request by lyrra**
- [x] I signed the [CLA](https://musescore.org/en/cla)

Needed for PR https://github.com/musescore/MuseScore/pull/19246

### CODE REVIEW

**Comment by Eism** [comment]
If you want to break the PR into small parts and thereby solve the problem with a stuck PR, then you should break it down into PRs that solve some problem. 
In this PR, I see that you have added `remoteSeek` and `remotePlayOrStop`, but do not use them anywhere. This approach is bad because if you don’t want to continue porting, then this non-working code will remain.

File: src/framework/midi/miditypes.h [hunk 1]
```diff
@@ -135,8 +135,6 @@ struct MidiDevice {
     }
 };
 
-using MidiDeviceList = std::vector<MidiDevice>;
```
Comment (Eism) [inline src/framework/midi/miditypes.h:138]: can you explain why you decided to delete the alias?

Comment (Eism) [reply src/framework/midi/miditypes.h:138]: can you explain in more detail? because I don't agree with you in this case

Comment (lyrra) [reply src/framework/midi/miditypes.h:138]: It isn't a list but a vector. And the alias makes it opaque, you dont know what methods you can use on it.
Anyway this commit is purely cosmetic, so I can withdraw it.

Comment (Eism) [reply src/framework/midi/miditypes.h:138]: yes, please

**Comment by lyrra** [comment]
> If you want to break the PR into small parts and thereby solve the problem with a stuck PR, then you should break it down into PRs that solve some problem. In this PR, I see that you have added `remoteSeek` and `remotePlayOrStop`, but do not use them anywhere. This approach is bad because if you don’t want to continue porting, then this non-working code will remain.

I've removed that commit, so this PR should be good to go!

**Comment by Jojo-Schmitz** [comment]
Why closing?

**Pull request closed without merging**